### ISSUE TITLE
Use shorter status message to relay changes to chart/table

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#910]
@brockfanning Converting to issue so that we can discuss DAC's response

DAC's consultancy response:

> The reason it is recommended to place it closer to the filters is that low vision users who require a high magnification may not notice this message – two of our low vision users did not locate the message when it appeared. However, as they are still able to select the filters and the data still appears in the table, I would agree that the message only needs to be relayed when users actually go to view the table – in which case, they should locate the message, so I would say this is fine.
 
Side note relating to DAC_On_Input_01, I just noticed with JAWS and every time a new filter is selected, all of the selected sub-categories are relayed as a status message. I would strongly advise that only the current change should be relayed. i.e. when age group ’65 to 74’ is selected, instead of:

**Current Code Ref(s):** #indicator-data-view-status

```
<span role="status" class="sr-only" id="indicator-data-view-status">Chart and table shows datasets for United Kingdom and Female and Male and 15 and under and 16 to 24 and 25 to 49 and 65 to 74 and 15 and under, Female and 16 to 24, Female and 25 to 49, Female and 65 to 74, Female and 15 and under, Male.</span>
```

Please make sure that status message just relays something like:

```
<span role="status" class="sr-only" id="indicator-data-view-status">Chart and table updated to include breakdows for age group 65 to 74</span>
```

- otherwise it becomes confusing.

**Comment by LucyGwilliamAdmin** [comment]
@brockfanning what are your thoughts on DAC's response to this one?

**Comment by brockfanning** [comment]
I believe this should be possible - it's a bit tricky technically but I don't think impossible.
